Sources/MapboxDirections/Intersection.swift

```diff
@@ -160,12 +168,21 @@ extension Intersection: Codable {
         case outletRoadClasses = "classes"
         case tollCollection = "toll_collection"
         case tunnelName = "tunnelName"
+        case mapboxStreets = "mapbox_streets_v8"
         case isUrban = "is_urban"
         case restStop = "rest_stop"
         case administrativeRegionIndex = "admin_index"
         case geometryIndex = "geometry_index"
     }
     
+    /// Used to code `Intersection.streetClass`
+    private struct MapboxStreetClassCodable: Codable {
+        private enum CodingKeys: String, CodingKey {
+            case streetClass = "class"
+        }
+        
+        let streetClass: MapboxStreetClass?
+    }
```

Comment:
Introduced this container struct for coding purposes. Since there is not other data available under `mapbox_streets_v8` object, I see this as most effective solution. If in the future more data will be stored under this key - it will be trivial to extract this into separate entity

Reply:
I’m pretty sure a new road class value would wait until a Mapbox Streets source v9, because it would be pretty disruptive to existing styles.